doc/character_selector.rdoc

```diff
@@ -0,0 +1,64 @@
+== Character Selectors
+
+A _character_ _selector_ is a string argument accepted by certain Ruby methods.
+Each of these instance methods accepts one or more character selectors:
+
+- String#tr(selector, replacements): returns a new string.
+- String#tr!(selector, replacements): returns +self+.
```

Comment:
#5626 called the method parameters `replaceables` and `replacements`, this should be consistent and use the same names.

Reply:
When this PR is merged, #tr revision will be next -- unless you think it should be added here.